src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -237,8 +237,11 @@ private void sendBlocks(String url, List<LayoutBlock> layoutBlocks) {
         try {
             WebhookResponse response = slack.send(url, payload);
             log.info("Send the latest user blocks to slack with response code " + response.getCode());
+            if (response.getCode() > 200) {
```

Comment:
Is != better? It wouldn't really make sense to send any response code <200, but figured != might be better. If not, feel free to disregard.

Reply:
Yeah, I think it makes sense.